web/src/app/modules/notification/modules/modals/services/confirmation-modal.service.ts

```diff
@@ -16,18 +16,18 @@ export class ConfirmationModal {
     }
 
     public confirmDelete(title: string, message: string): Promise<any> {
-        const modalRef = this.modalService.open(TypedModalContent);
-        modalRef.componentInstance.options = Dialogtype.okCancelDialog(title, message);
-        return modalRef.result;
+        return this.change(title,message);
     }
 
     public openOk(title: string, message: string): Promise<any> {
         const modalRef = this.modalService.open(TypedModalContent);
         modalRef.componentInstance.options = Dialogtype.okDialog(title, message);
         return modalRef.result;
     }
-
-    public openOkCancel(title: string, message: string): Promise<any> {
+  public openOkCancel(title: string, message: string): Promise<any> {
+        return this.change(title,message);
+    }
+    public change(title: string, message: string): Promise<any> {
```

Comment:
This method name needs to be more informative. Change to openOkCancel.
Also, there is no need for this method to be public.